source/module_relax/relax_driver.cpp

```diff
@@ -115,6 +123,15 @@ void Relax_Driver<FPTYPE, Device>::relax_driver(ModuleESolver::ESolver *p_esolve
                     GlobalV::ofs_running << "\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%" << std::endl;
                 }
             }
+#ifdef __RAPIDJSON
+            //add Json of cell coo stress force
+            double unit_transform = ModuleBase::RYDBERG_SI / pow(ModuleBase::BOHR_RADIUS_SI, 3) * 1.0e-8;
+            double fac = ModuleBase::Ry_to_eV / 0.529177;
+            Json::add_output_cell_coo_stress_force(
+                &GlobalC::ucell,
+                force,fac,
+                stress,unit_transform);
+#endif //__RAPIDJSON 
```

Comment:
Here, should be outer of ` if (GlobalV::CALCULATION == "scf" || GlobalV::CALCULATION == "relax" || GlobalV::CALCULATION == "cell-relax")`